routes/api-v1.php

```diff
@@ -24,3 +25,4 @@
 
 Route::post('generate-signed-url', SignedUrlController::class);
 Route::get('countries', CountryController::class);
+Route::apiResource('versions', MasterSettingController::class)->only(['index', 'show']);
```

Comment:
@damini-7span Please add this route in admin.php route file, And it's access after the login.

Reply:
As we discussed we decided to take is_public field in database table and remain route in api-v1.php as it is.